### ISSUE TITLE
Test fail with python38-flask 2.01

### ISSUE THREAD

**Issue by coogor** [onionshare/onionshare#1385]
Hi,
looking at the build result in https://build.opensuse.org/package/show/devel:languages:python/python-onionshare with a recent version of python-flask it runs into errors, see attached log from line 2300 onwards.
With an older verison of flask the tests are fine (see build result for Leap 15.2)

Is this an incompatibility in onionshare?

Please advise!
Thank you!
[testresult.tar.gz](https://github.com/micahflee/onionshare/files/6847306/testresult.tar.gz)

**Comment by djerun** [comment]
```
Traceback (most recent call last):
  File "/home/user/env/bin/onionshare-cli", line 8, in <module>
    sys.exit(main())
  File "/home/user/env/lib/python3.9/site-packages/onionshare_cli/__init__.py", line 326, in main
    web = Web(common, False, mode_settings, mode)
  File "/home/user/env/lib/python3.9/site-packages/onionshare_cli/web/web.py", line 94, in __init__
    self.generate_static_url_path()
  File "/home/user/env/lib/python3.9/site-packages/onionshare_cli/web/web.py", line 192, in generate_static_url_path
    self.app.add_url_rule(
  File "/home/user/env/lib/python3.9/site-packages/flask/scaffold.py", line 56, in wrapper_func
    return f(self, *args, **kwargs)
  File "/home/user/env/lib/python3.9/site-packages/flask/app.py", line 1092, in add_url_rule
    raise AssertionError(
AssertionError: View function mapping is overwriting an existing endpoint function: static

```

[onionshare_cli/web/web.py](https://github.com/micahflee/onionshare/blob/d4d6eea500feb825d5639354b8aad1be6ba451f5/cli/onionshare_cli/web/web.py#L192)
```
self.app.add_url_rule(
            self.static_url_path + "/<path:filename>",
            endpoint="static",
            view_func=self.app.send_static_file,
        )
```

`diff [1.1](https://flask.palletsprojects.com/en/1.1.x/api/#flask.Flask.add_url_rule) [2.0](
https://flask.palletsprojects.com/en/2.0.x/api/#flask.Flask.add_url_rule)`
```
[...]
<     If the view_func is not provided you will need to connect the endpoint to a view function like so:
---
>     The endpoint name for the route defaults to the name of the view function if the endpoint parameter isn’t passed. An error will be raised if a function has already been registered for the endpoint.
[...]
```

**Comment by mig5** [comment]
This got fixed as part of a PR for #1365 so I'm going to close this. Thanks for the report
